en/docs/fhir/guides/connecting-ehr-emr-systems.md

```
@@ -18,10 +18,10 @@ Follow the instructions given below to set up the connectors you require in your
 <li>    The connector zip file will be provided after it is generated by the tool.</li>
 </ul>
 
-##Install the connector
+## Install the connector
 As mentioned in the previous section, WSO2 Integration Studio is the IDE that will be used to develop integrations that will be deployed in the integration layer of the Healthcare solution. The connector zip that is obtained from WSO2 should be imported to the Integration Studio in order to develop the integration flow.<br /> Steps on installing and using the connector can be found by clicking the [link](https://apim.docs.wso2.com/en/4.0.0/reference/connectors/connectors-overview/#how-to-use-connectors)<br />
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)